### PR TITLE
[JENKINS-65398] Terminology update

### PR DESCRIPTION
**Pull request by twasyl**
Terminology update as part of [JENKINS-65398](https://issues.jenkins.io/browse/JENKINS-65398).

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/config.jelly [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 
-    <f:entry title="Mesos Master" field="mesosMasterUrl">
+    <f:entry title="Mesos Controller" field="mesosMasterUrl">
```
Comment (Vlatombe) [inline src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/config.jelly:5]: ditto

File: src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/help-mesosMasterUrl.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div>
-  Set the Mesos master address in following format:
+  Set the Mesos controller address in following format:
   <ul>
-    <li><b>mesos_hostname:mesos_port</b> if your setup only have one Mesos master</li>
-    <li><b>zk://mesos_1:zookeeper_port,mesos_2:zookeeper_port,mesos_3:zookeeper_port/mesos</b> if your setup is a multi master node Mesos cluster using Zookeeper</li>
+    <li><b>mesos_hostname:mesos_port</b> if your setup only have one Mesos controller</li>
```
Comment (Vlatombe) [inline src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/help-mesosMasterUrl.html:4]: ditto

File: src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/help-mesosMasterUrl.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+    <li><b>zk://mesos_1:zookeeper_port,mesos_2:zookeeper_port,mesos_3:zookeeper_port/mesos</b> if your setup is a multi controller node Mesos cluster using Zookeeper</li>
```
Comment (Vlatombe) [inline src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/help-mesosMasterUrl.html:5]: ditto

File: src/test/java/org/jenkinsci/plugins/mesos/JenkinsConfigClient.java [hunk 4]
```diff
@@ -32,7 +32,7 @@ public JenkinsConfigClient(WebClient jenkinsClient) throws IOException {
   /**
    * Submits a Jenkins configuration form and adds a Mesos Cloud with one agent specs.
    *
-   * @param mesosMasterUrl The URL of the Mesos master to connect to.
+   * @param mesosMasterUrl The URL of the Mesos controller to connect to.
```
Comment (Vlatombe) [inline src/test/java/org/jenkinsci/plugins/mesos/JenkinsConfigClient.java:35]: ditto

File: src/test/java/org/jenkinsci/plugins/mesos/MesosCloudTest.java [hunk 5]
```diff
@@ -26,7 +26,7 @@ void deserializeOldConfig(TestUtils.JenkinsRule j) throws IOException {
                 "config_1.x.xml",
                 StandardCharsets.UTF_8,
                 Thread.currentThread().getContextClassLoader())
-            // Master URL resolution requires a separate test.
+            // Controller URL resolution requires a separate test.
```
Comment (Vlatombe) [inline src/test/java/org/jenkinsci/plugins/mesos/MesosCloudTest.java:29]: ditto

**Review by Vlatombe — APPROVED** [review]

**Review by aHenryJard — APPROVED** [review]

**Comment by jeschkies** [comment]
@twasyl thanks for doing this. Would you mind applying `gradlew spotlessApply` to format the code so that the build passes?

**Comment by twasyl** [comment]
> @twasyl thanks for doing this. Would you mind applying `gradlew spotlessApply` to format the code so that the build passes?

@jeschkies Yes sure, I've done that and pushed

**Pull request merged**